samples/dashboards/sync/datacursor-sync/demo.html

```diff
@@ -5,6 +5,10 @@
 
 <div id="container"></div>
 <p class="highcharts-description">
-  This demo shows the DataCursor visual states like the highlight on points. You 
-  can hover on a point on a chart and related point on another chart will be highlighed.
+  This dashboard demonstrates the usage of DataCursor for visual states like the
+  highlight on points. You can hover over a point in the charts and related
+  points on other charts will be highlighed as well. The data shows the amount
+  of world wide production in million metric tons for certain vegetables taken
+  from
+  <a href="https://en.wikipedia.org/wiki/Vegetable#Some_common_vegetables">Wikipedia</a>.
```

Comment:
Would be good to add `target`, `noreferal` and `title` params.

Reply:
What should be the target?

Reply:
`target="_blank"`

Reply:
This is for accessibility reasons forbidden because a new tab/window detaches the user from their current browsing history.